### ISSUE TITLE
Fits don's include central value

### ISSUE THREAD

**Issue by lonbar** [nikosarcevic/Derivative-Calculator#4]
One thing I noticed is that the `x_values` used to compute the function values don't include the central value (see [derivative_calculator.py at lines 55–66](https://github.com/nikosarcevic/Derivative-Calculator/blob/b332b9b61083e38a516a25a9aea58a8b128c53fa/derivative_calculator.py#L55-L66)). This isn't difficult to fix and may in fact not be necessary, but it is good to be aware of this.

**Comment by nikosarcevic** [comment]
> One thing I noticed is that the `x_values` used to compute the function values don't include the central value (see [derivative_calculator.py at lines 55–66](https://github.com/nikosarcevic/Derivative-Calculator/blob/b332b9b61083e38a516a25a9aea58a8b128c53fa/derivative_calculator.py#L55-L66)). This isn't difficult to fix and may in fact not be necessary, but it is good to be aware of this.

yeah I noticed it, probs because I was adding the gaussian noise to it. can fix it when I do the upgrade with the broadcasting etc
